x/leverage/keeper/grpc_query.go

```diff
@@ -94,6 +94,29 @@ func (q Querier) Borrowed(
 	return &types.QueryBorrowedResponse{Borrowed: sdk.NewCoins(token)}, nil
 }
 
+func (q Querier) AvailableBorrow(
+	goCtx context.Context,
+	req *types.QueryAvailableBorrowRequest,
+) (*types.QueryAvailableBorrowResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+	if req.Denom == "" {
+		return nil, status.Error(codes.InvalidArgument, "invalid denom")
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	if !q.Keeper.IsAcceptedToken(ctx, req.Denom) {
+		return nil, status.Error(codes.InvalidArgument, "not accepted Token denom")
+	}
+
+	// Available for borrow = Module Balance - Reserve Amount
+	moduleBalance := q.Keeper.ModuleBalance(ctx, req.Denom)
+	reserveAmount := q.Keeper.GetReserveAmount(ctx, req.Denom)
```

Comment:
nit: can we actually put this into a method on the `keeper` type? You never know where else we might need this.

Reply:
done at https://github.com/umee-network/umee/pull/379/commits/5692bbea45ec0508050569f9a4fc947f731872b2